From 04462b82092b7388803bb652a768927fdeda357f Mon Sep 17 00:00:00 2001
From: Rob Evers <revers@redhat.com>
Date: Mon, 23 Jun 2014 21:41:50 -0400
Subject: [scsi] lpfc 10.2.8000.0: Fix dynamic transitions of FirstBurst from on to off

Message-id: <1403559726-20488-9-git-send-email-revers@redhat.com>
Patchwork-id: 84427
O-Subject: [RHEL7.1 e-stor PATCH 08/24] scsi: lpfc 10.2.8000.0: Fix dynamic transitions of FirstBurst from on to off
Bugzilla: 1088574
RH-Acked-by: Ewan Milne <emilne@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Dick Kennedy <rkennedy@Emulex.Com>

Submitted by James Smart James.Smart@emulex.com

Fix dynamic transitions of FirstBurst from on to off

Signed-off-by: James Smart james.smart@emulex.com

http://marc.info/?l=linux-scsi&m=139663388921848&w=2
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/lpfc/lpfc_scsi.c b/drivers/scsi/lpfc/lpfc_scsi.c
index 462453e..4adeb21 100644
--- a/drivers/scsi/lpfc/lpfc_scsi.c
+++ b/drivers/scsi/lpfc/lpfc_scsi.c
@@ -4314,6 +4314,7 @@ lpfc_scsi_prep_cmnd(struct lpfc_vport *vport, struct lpfc_scsi_buf *lpfc_cmd,
   fcp_cmnd->fcpCntl1 = SIMPLE_Q;
 
  sli4 = (phba->sli_rev == LPFC_SLI_REV4);
+ piocbq->iocb.un.fcpi.fcpi_XRdy = 0;
 
  /*
   * There are three possibilities here - use scatter-gather segment, use
-- 
1.7.1